docs/quick-start-guide.md

```diff
@@ -110,7 +110,9 @@ To run the Spark Pi example, run the following command:
 $ kubectl apply -f examples/spark-pi.yaml
 ```
 
-This will create a `SparkApplication` object named `spark-pi`. Check the object by running the following command:
+Note that `spark-pi.yaml` configures the driver pod to use the `spark` service account to communicate with the Kubernetes API server. You might need to replace it with the approprate service account before submitting the job. If you installed the operator using the Helm chart, the Spark job namespace (i.e. `default` by default) already has a service account you can use. Its name ends with `-spark` and starts with the Helm release name. 
```

Comment:
I think it's worth mentioning the Helm char parameters (`createSparkJobNamespace` and `sparkJobNamespace`) here to go with `the Spark job namespace`.

Reply:
Added